client/my-sites/earn/memberships/products.jsx

```diff
@@ -322,6 +343,109 @@ class MembershipsProductsSection extends Component {
 						) }
 					</FormToggle>
 				</FormFieldset>
+			</>
+		);
+	}
+
+	renderEmailTab() {
+		const { translate } = this.props;
+		const { editedPostsEmail, editedCustomConfirmationMessage } = this.state;
+		return (
+			<>
+				<FormFieldset>
+					<h6 className="memberships__dialog-form-header">{ translate( 'Posts via email' ) }</h6>
+					<p>
+						{ translate(
+							'Allow members of this recurring payment plan to opt into receiving new posts via email.'
+						) }{ ' ' }
+						<a href="/TODO">{ translate( 'Learn more' ) }.</a>
+					</p>
+					<FormToggle
+						onChange={ ( newValue ) => this.setState( { editedPostsEmail: newValue } ) }
+						checked={ editedPostsEmail }
+					>
+						{ this.props.translate(
+							'Email newly published posts to members of this recurring payment plan who have opted in.'
+						) }
+					</FormToggle>
+				</FormFieldset>
+				<FormFieldset>
+					<h6 className="memberships__dialog-form-header">
+						{ translate( 'Custom confirmation message' ) }
+					</h6>
+					<p>
+						{ translate(
+							'Add a custom message to the confirmation email that is sent out for this recurring payment plan.'
+						) }
+					</p>
+					<FormTextArea
+						value={ editedCustomConfirmationMessage }
+						onChange={ ( event ) =>
+							this.setState( { editedCustomConfirmationMessage: event.target.value } )
+						}
+					/>
+				</FormFieldset>
+			</>
+		);
+	}
+
+	renderEditDialog() {
+		return (
+			<Dialog
+				className="memberships__dialog"
+				isVisible={ this.state.showDialog }
+				onClose={ this.onCloseDialog }
+				buttons={ [
+					{
+						label: this.props.translate( 'Cancel' ),
+						action: 'cancel',
+					},
+					{
+						label: this.state.editedProductId
+							? this.props.translate( 'Edit' )
```

Comment:
I know this was in there originally but should this button label be `Save`?

Reply:
+1 to @eoigal's suggestion to update this label to Save or Save Changes. We can also update the title from edit to "Edit Recurring Payment Plan". Depending on space we could maybe add the description or amount to the header so folks know which one they're editing.

| Add | Edit |
| ------------- | ------------- |
|  <img width="816" alt="Screen Shot 2020-05-20 at 3 54 50 PM" src="https://user-images.githubusercontent.com/1270189/82505414-87b71400-9ab2-11ea-979c-b9e0a0fc241b.png"> | <img width="858" alt="Screen Shot 2020-05-20 at 3 54 33 PM" src="https://user-images.githubusercontent.com/1270189/82505384-766e0780-9ab2-11ea-882c-183216ab8002.png"> |

Reply:
> I know this was in there originally but should this button label be Save?

a46547a

> We can also update the title from edit to "Edit Recurring Payment Plan"

00c58db

> Depending on space we could maybe add the description or amount to the header

That leads to other question: if the uses changes one of them, should we update the header as well before the user clicks on "Save"? I'm not sure what's better, so I used a "Edit Recurring Payment Plan" static title for now.